### PR TITLE
Honors and Awards section of the new wechaty landing page (Section 5)

### PR DESCRIPTION
**Pull request by sajen-k**
Issue Number: #912  

This PR comprises of an atomic PR of fifth section (Honors and Awards) for the new wechaty landing page.

Below are the images of Honors and Awards Section running locally

Desktop Version:
Screenshot 2021-09-08 at 9 04 36 PM

Mobile Version:
Screenshot 2021-09-08 at 9 05 43 PM
Screenshot 2021-09-08 at 9 05 53 PM

### CODE REVIEW

File: docusaurus/src/pages/honors-and-awards-section/honors-and-awards-section.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import styles from "./honors-and-awards-section.module.css";
+
+const honorsAndAwards = () => {
+    
+    return(
+        <div className={styles.body}>
```
Comment (huan) [inline docusaurus/src/pages/honors-and-awards-section/honors-and-awards-section.tsx]: The `HonorsAndAwardsSection` looks very good!

However, I saw you add custom css on every line of the HTML code, which is very verbose.

Can we apply CSS once for the whole react html5 component?

**Review by huan — CHANGES_REQUESTED** [review]

File: docusaurus/src/pages/honors-and-awards-section/honors-and-awards-section.tsx (hunk 1 above)
Comment (sajen-k) [reply docusaurus/src/pages/honors-and-awards-section/honors-and-awards-section.tsx]: Done, I have made changes in all the PRs as you said

**Review by huan — DISMISSED** [review]
I saw all CSS are using `id` (`#id`), and every HTML section has different `id`s.

I think we should use a more general class to specify this?

For example, `honorsAndAwardsFirstWrapper` and `honorsAndAwardsSecondWrapper` seems not necessary to be different?

**Comment by sajen-k** [comment]
> I saw all CSS are using `id` (`#id`), and every HTML section has different `id`s.
> 
> I think we should use a more general class to specify this?
> 
> For example, `honorsAndAwardsFirstWrapper` and `honorsAndAwardsSecondWrapper` seems not necessary to be different?

@huan ok I will change it accordingly.

**Review by Rohitesh-Kumar-Jain — APPROVED** [review]

**Review by proudofsimin — APPROVED** [review]

**Review by arnabiscreating — APPROVED** [review]

**Pull request merged**